### ISSUE TITLE
Exporting the xy-coordinates of the manhattan plot as a text file

### ISSUE THREAD

**Issue by scikal** [abyzovlab/CNVpytor#83]
Hi, 

CNVpytor in a valuable tool in my arsenal --- very powerful but in the same time elegant and easy to use.

I was wondering if there is a simple way to export the xy-coordinates of the manhattan plot as a text file via the command line.

Thanks in advance,
Daniel

**Comment by suvakov** [comment]
Thank you. 

To export data you can create python script. For manhattan plot, save following script to a file, e.g. "export_rd.py":
```
import cnvpytor
file = "file.pytor"
bin_size = 100000
flag = cnvpytor.FLAG_GC_CORR
view = cnvpytor.Viewer([file],{"bin_size":bin_size})
chroms = []
for c, (l, t) in view.reference_genome["chromosomes"].items():
    rd_chr = view.io[0].rd_chromosome_name(c)
    if (cnvpytor.Genome.is_autosome(c) or cnvpytor.Genome.is_sex_chrom(c)):
        chroms.append((rd_chr, l))

for c, l in chroms:
    rd = view.io[0].get_signal(c,bin_size,"RD",flag)
    for i,x in enumerate(rd):
        print("%s\t%d\t%.0f" % (c,i*bin_size,x)) 
```
Ajust file name, bin size and flag (for masked rd use `flag = cnvpytor.FLAG_GC_CORR | cnvpytor.FLAG_USEMASK`) and run by
```
> python export_rd.py
```
or
```
> python3 export_rd.py
```
It should print out three columns: chromosome, bin starting position, value of RD.

If you need only data for a single chromosome, there simpler one-line solution:
```
echo '|np.savetxt("test.txt",self.io[0].get_signal("chr1", 100000, "RD",FLAG_GC_CORR))' | cnvpytor -root file.pytor -view 100000
```
It will store RD as single column txt file 'test.txt'.

**Comment by xiucz** [comment]
@suvakov 
Thanks for your code here, after running it for my low pass WGS cnv data, I want why the value of the third column is larger than 1x ~10x ? And the mangattan plot y axis values are as expected. 
![image](https://user-images.githubusercontent.com/12706146/164188663-9ee0685e-54a1-4f53-9e93-02f4d73862ae.png)

**Comment by fsnibs10** [comment]
Hi, 

Thanks for giving the python script @suvakov . I tried to generate the values for my sequencing samples. It promoted the error information as below. One possible reason is from the non-human species.  Before I had constructed the genome config file "pombe_ref_genome_conf.py" for running cnvpytor command successfully. Now I want to export the values from  manhattan plot . How can I add genome parameters in python script?  Many thanks!

`>>> import cnvpytor
>>> file="DY48288.pytor"
>>> bin_size = 500
>>> flag = cnvpytor.FLAG_GC_CORR
>>> view = cnvpytor.Viewer([file],{"bin_size":bin_size})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/data/a/fs/anaconda3/envs/cnvpytor/lib/python3.7/site-packages/cnvpytor/viewer.py", line 363, in __init__
    self.reference_genome = Genome.reference_genomes[rg_name]
KeyError: 'pombe'
`
